profiles/coreos/base/package.use

```diff
@@ -39,7 +39,7 @@ net-analyzer/nmap ncat -lua
 app-admin/sudo -sendmail
 
 # avoid pulling in gnutls, disable gentoo-only bits, enable journal upload
-sys-apps/systemd -ssl curl vanilla lzma
+sys-apps/systemd -ssl curl vanilla -lz4 lzma gcrypt
```

Comment:
Ok, let's stick with lzma.
On Feb 21, 2015 2:30 PM, "Alex Crawford" <notifications@github.com> wrote:

> In profiles/coreos/base/package.use
> <https://github.com/coreos/coreos-overlay/pull/1113#discussion_r25126537>:
>
> > @@ -39,7 +39,7 @@ net-analyzer/nmap ncat -lua
> >  app-admin/sudo -sendmail
> >
> >  # avoid pulling in gnutls, disable gentoo-only bits, enable journal upload
> > -sys-apps/systemd -ssl curl vanilla lzma
> > +sys-apps/systemd -ssl curl vanilla -lz4 lzma gcrypt
>
> I noticed that if I enable both, journald compresses with lz4 by default.
> Fedora, because it only has lzma, uses XZ by default. So, CoreOS will be
> able to read Fedora journals, but not the other way around.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/coreos-overlay/pull/1113/files#r25126537>.
>